app/src/main/java/app/gamenative/ui/screen/login/UserLoginScreen.kt

```diff
@@ -419,6 +434,33 @@ private fun ModernUsernamePassword(
                 .fillMaxWidth()
                 .padding(bottom = 16.dp)
         ) {
+            if (!isSteamConnected) {
```

Comment:
ditto here - we should only surface this if an attempt was made to steam and it failed, otherwise it could be confusing. Is that how it's working?